Modules/readline.c

```diff
@@ -1033,10 +1039,14 @@ on_startup_hook(void)
 static int
 #if defined(_RL_FUNCTION_TYPEDEF)
 on_pre_input_hook(void)
+{
+#elif defined(__APPLE__) && defined(WITH_APPLE_EDITLINE)
```

Comment:
`__APPLE__` is implied if `WITH_APPLE_EDITLINE` is defined, so this should suffice:

```suggestion
#elif defined(WITH_APPLE_EDITLINE)
```

Reply:
Also, keeping the opening brace out of the ifdef should result in a cleaner diff?